client/my-sites/plans/current-plan/index.jsx

```diff
@@ -29,13 +29,19 @@ import { getPlan } from 'lib/plans';
 import QuerySiteDomains from 'components/data/query-site-domains';
 import { getDecoratedSiteDomains, isRequestingSiteDomains } from 'state/sites/domains/selectors';
 import DomainWarnings from 'my-sites/upgrades/components/domain-warnings';
+import isSiteAutomatedTransfer from 'state/selectors/is-site-automated-transfer';
 
 class CurrentPlan extends Component {
 	static propTypes = {
 		selectedSiteId: PropTypes.number,
 		selectedSite: PropTypes.object,
 		isRequestingSitePlans: PropTypes.bool,
-		context: PropTypes.object
+		context: PropTypes.object,
+		domains: PropTypes.array,
+		currentPlan: PropTypes.object,
+		isExpiring: PropTypes.bool,
+		shouldShowDomainWarnings: PropTypes.bool,
```

Comment:
I don't think it's needed since`shouldShowDomainWarnings` is a computed prop in the connect function.